firebase-perf/dev-app/dev-app.gradle

```diff
@@ -83,7 +83,7 @@ dependencies {
     if (project.hasProperty("fireperfBuildForAutopush")) {
         println("Building with HEAD version ':firebase-perf' of SDK")
         implementation project(":firebase-perf")
-        implementation project(":firebase-common")
+        implementation "com.google.firebase:firebase-common:20.3.1"
```

Comment:
maybe in this case it would make sense to leave it as project level dependency? not sure if it will change how the test app works

Reply:
Done